README.md

```diff
@@ -1,70 +1,106 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Final-Capstone-Front-End Project
 
-## Available Scripts
+> In this task, we used the live API data from the [Final-Capstone-Back-End Project](https://github.com/andersonlebon/Final-Capstone-backend).
 
-In the project directory, you can run:
+The objective was to build a house booking application(with discount) where the user has the posibilities to add, reserve and delete houses with added advantage to view house details.The application has authentication whereby the logged in user can only delete the houses that he/she added.
 
-### `yarn start`
+## Screenshots
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+#### Login auth page
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+![login](login.png)
 
-### `yarn test`
+#### Main Page
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+![Main_page](Main_page.png)
 
-### `yarn build`
+#### Remove House Page
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+![removehouse](removehouse.png)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+#### Reservation Page
+![Reservation](Reservation.png)
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+#### Mobile version
+![mobileV](mobileV.png)
+![mobile](mobile.png)
 
-### `yarn eject`
+## Built With
+- React-Redux
+- HTML/SCSS
+- [Final-capstone-Back-End API](https://house-booking-api.herokuapp.com/api-docs/index.html)
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Live Demo
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+Find the application's live version [HERE]()
```

Comment:
- [ ] Just so you don't forget to update this link 😄

Reply:
Thank you for the reminder. We decided to deploy it at the end after the reviews.